.github/workflows/ci.yml

```diff
@@ -98,6 +108,7 @@ jobs:
           summary_output_file: results.json
           summary_display: true
           baseline_file: baseline.json
+          include: "to-scan/alfresco-transform*"
```

Comment:
Why did we have to add this? Won't this mess up the detection logic for other libraries that don't start with `alfresco-transform` such as e.g.: `alfresco-base-t-engine-*.jar` that is mentioned [within this very workflow file](https://github.com/Alfresco/alfresco-transform-core/pull/956/files#diff-b803fcb7f17ed9235f1e5cb1fcd2f5d3b2838429d4368ae4c57ce4436577f03fR88).

Reply:
The -include parameter only allows to choose from what already is recognized by SAST Scan as top module. In this case, this allows us to remove the jwarc.jar . 
If you think this is unnecessary, I can revert it.

Here is the effect of this line:

==========================
Found 2 Scannable modules.
==========================
JS files within jwarc-0.29.0.jar
to-scan/alfresco-transform-core-aio-5.1.2-A2-SNAPSHOT.jar

===================
Analyzed 1 modules.
===================
to-scan/alfresco-transform-core-aio-5.1.2-A2-SNAPSHOT.jar

Reply:
We can achieve the very same result by having a more lax include filter such as `to-scan/alfresco-*` then. Let's try that instead.